internal/locate/region_cache.go

```diff
@@ -133,6 +134,20 @@ func nextTTL(ts int64) int64 {
 	return ts + regionCacheTTLSec + jitter
 }
 
+var pdRegionMetaCircuitBreaker = circuitbreaker.NewCircuitBreaker(
```

Comment:
Shall we move it to `config`? @MyonKeminta

Reply:
Fine to me.
But if you are not going to add some fields to the config file, I think maybe it would be better to add another single file in the `internal/locate` directory for holding this variable.

Reply:
Nevermind. I just feel it still makes me kind of incomfortable if it's put in the `config` package while nothing of it is configurable by config file. Now it looks fine to me.